src/containerapp/azext_containerapp/_help.py

```diff
@@ -0,0 +1,495 @@
+    - name: Show the details of a container app.
+      text: |
+          az containerapp show -n MyContainerapp -g MyResourceGroup
+"""
+
+helps['containerapp list'] = """
+    type: command
+    short-summary: List container apps.
+    examples:
+    - name: List container apps in the current subscription.
+      text: |
+          az containerapp list
+    - name: List container apps by resource group.
+      text: |
+          az containerapp list -g MyResourceGroup
+"""
+
+# Revision Commands
+helps['containerapp revision'] = """
+    type: group
+    short-summary: Commands to manage revisions.
+"""
+
+helps['containerapp revision show'] = """
+    type: command
+    short-summary: Show details of a revision.
+    examples:
+    - name: Show details of a revision.
+      text: |
+          az containerapp revision show -n MyContainerapp -g MyResourceGroup \\
+              --revision MyContainerappRevision
+"""
+
+helps['containerapp revision list'] = """
+    type: command
+    short-summary: List a container app's revisions.
+    examples:
+    - name: List a container app's revisions.
+      text: |
+          az containerapp revision list -n MyContainerapp -g MyResourceGroup
+"""
+
+helps['containerapp revision restart'] = """
+    type: command
+    short-summary: Restart a revision.
+    examples:
+    - name: Restart a revision.
+      text: |
+          az containerapp revision restart -n MyContainerapp -g MyResourceGroup --revision MyContainerappRevision
+"""
+
+helps['containerapp revision activate'] = """
+    type: command
+    short-summary: Activate a revision.
+    examples:
+    - name: Activate a revision.
+      text: |
+          az containerapp revision activate -g MyResourceGroup --revision MyContainerappRevision
+"""
+
+helps['containerapp revision deactivate'] = """
+    type: command
+    short-summary: Deactivate a revision.
+    examples:
+    - name: Deactivate a revision.
+      text: |
+          az containerapp revision deactivate -g MyResourceGroup --revision MyContainerappRevision
+"""
+
+helps['containerapp revision set-mode'] = """
+    type: command
+    short-summary: Set the revision mode of a container app.
+    examples:
+    - name: Set a container app to single revision mode.
+      text: |
+          az containerapp revision set-mode -n MyContainerapp -g MyResourceGroup --mode Single
+"""
+
+helps['containerapp revision copy'] = """
+    type: command
+    short-summary: Create a revision based on a previous revision.
+    examples:
+    - name: Create a revision based on the latest revision.
+      text: |
+          az containerapp revision copy -n MyContainerapp -g MyResourceGroup \\
+              --cpu 0.75 --memory 1.5Gi
+    - name: Create a revision based on a previous revision.
+      text: |
+          az containerapp revision copy -g MyResourceGroup \\
+              --from-revision PreviousRevisionName --cpu 0.75 --memory 1.5Gi
+
+"""
+
+helps['containerapp revision copy'] = """
```

Comment:
Please double check.
Why two `containerapp revision copy` here ?

Reply:
@wangzelin007 thanks for pointing this out - but looks like some tests in "Verify Ref Docs" is still failing :(